README.md

```diff
@@ -2,24 +2,115 @@
 
 [![License](https://img.shields.io/badge/license-MIT-blue)](https://github.com/libsql/libsql-client-go/blob/main/LICENSE)
 
-This is the source repository of the Go SDK for libSQL. You can either connect to a local SQLite/libSQL database (embedded in the client) or to a remote libSQL server.
+This module implements a libSQL driver for the standard Go [database/sql
+package]. You can use it to interact with the following types of databases:
 
-## Running examples
+- Local SQLite database files
+- [libSQL sqld] instances (including [Turso])
 
-Using HTTPS:
-```console
-go run -ldflags="-X 'main.dbUrl=https://<login>:<password>@<sqld db url>'" examples/sql/counter/main.go
```

Comment:
I think it would be good to keep the command to run the examples in the docs. It not only allows new visitors to try the examples but also serves as a documentation for us in the future @CodingDoug

Reply:
The examples aren't exactly in good shape either. That's another thing I was going to point out separately.  Could you make sure they are in a condition where they work out of the box with no errors?